docker-compose/envs/common-blockscout.env

```diff
@@ -1,11 +1,19 @@
 # DOCKER_TAG=
 SECRET_KEY_BASE=RMgI4C1HSkxsEjdhtGMfwAHfyT6CKWXOgzCboJflfSm4jeAlic52io05KB6mqzc5
+<<<<<<< HEAD
+ETHEREUM_JSONRPC_VARIANT=evm
+=======
 ETHEREUM_JSONRPC_VARIANT=erigon
+>>>>>>> origin/evm
```

Comment:
fix this

Reply:
I added the evm config. 
evm is slightly different from erigon